src/console/dbseed/domain/db-seed.service.ts

```diff
@@ -191,6 +197,38 @@ export class DbSeedService {
         this.logger.info(`Insert ${files.length} entities of type Rolle`);
     }
 
+    public async seedRollenerweiterung(fileContentAsStr: string): Promise<void> {
+        const rollenerweiterungFile: EntityFile<RollenerweiterungFile> = JSON.parse(
+            fileContentAsStr,
+        ) as EntityFile<RollenerweiterungFile>;
+        const files: RollenerweiterungFile[] = plainToInstance(RollenerweiterungFile, rollenerweiterungFile.entities);
+        for (const file of files) {
+            const orga: Organisation<true> = await this.getReferencedOrganisation(file.organisationId);
+            const rolle: Rolle<true> = await this.getReferencedRolle(file.rolleId);
+            const sp: ServiceProvider<true> = await this.getReferencedServiceProvider(file.serviceProviderId);
+            const rollenerweiterung: Rollenerweiterung<false> | DomainError = this.rollenerweiterungFactory.createNew(
```

Comment:
Creating the aggregate can't fail and with the annotation it looks like error-handling is missing.
```suggestion
            const rollenerweiterung: Rollenerweiterung<false> = this.rollenerweiterungFactory.createNew(
```